server/rucio.conf.j2

```diff
@@ -82,7 +82,9 @@ CacheRoot /tmp
  ErrorLog /dev/stderr
 {% endif %}
 
-{% if RUCIO_HTTPD_ENCODED_SLASHES|default('False') == 'True' %}
+{% if RUCIO_HTTPD_ENCODED_SLASHES_NO_DECODE|default('False') == 'True' %}
+ AllowEncodedSlashes NoDecode
+{% elif RUCIO_HTTPD_ENCODED_SLASHES|default('False') == 'True' %}
```

Comment:
I wonder if at least one of the two should be `True` by default?

Reply:
Yes, in the future I think we should just have

```
AllowEncodedSlashes NoDecode
```

as a default, or even as the only option.

There is another PR (https://github.com/rucio/containers/pull/388) to change the default behaviour, but since this might hve unintended consequences, I created this PR.